public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -496,14 +497,14 @@ table
 
 :marked
   ### Binding target
-  An element property between enclosing square brackets identifies the target property. The target property in the following code is the image element’s `src` property.
+  An element's target property is placed between enclosing square brackets. The target property in the following code is the image element’s `src` property.
 
 +makeExample('template-syntax/ts/app/app.component.html', 'property-binding-1')(format=".")
 :marked
-  Some people prefer the `bind-` prefix alternative, known as the *canonical form*:
+  Note, some people prefers the `bind-` prefix alternative, known as the *canonical form*:
```

Comment:
We usually avoid "note" unless we really want to make it a note (which we'd usually denote with a callout or alert).